### PR TITLE
V3 react doc gen

### PR DESCRIPTION
**Pull request by srsharon**

### CODE REVIEW

File: module/components/Divider/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /**
+     * Useful to extend the style applied to components.
+     */
+    className: PropTypes.string,
```
Comment (jeradrutnam) [inline module/components/Divider/index.jsx]: Please add other props as well.

File: module/index.js [hunk 2]
```diff
@@ -11,4 +11,5 @@ export { Chip } from './components/Chip';
 export { Grid } from './components/Grid';
 export { Paper } from './components/Paper';
 export { Typography } from './components/Typography';
+export { Divider } from './components/Divider';
```
Comment (dakshika) [inline module/index.js]: Why this after the Typo?

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/stylesheets/components/_divider.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+.divider {
+    border-top: 1px solid;
+  border-color: #333;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_divider.scss]: Fix Indentation and change these hard coded values to variables.

File: module/components/Divider/Readme.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Divider example:
+
+```js
+<Divider/>
+```
+
+Light Divider example:
+```js
+<Divider light="light" />
```
Comment (jeradrutnam) [inline module/components/Divider/Readme.md]: Why are we passing value "light" for property light

File: module/components/Divider/index.jsx (hunk 1 above)
Comment (srsharon) [reply module/components/Divider/index.jsx]: Done

File: module/components/Divider/Readme.md (hunk 4 above)
Comment (srsharon) [reply module/components/Divider/Readme.md]: Done

File: module/stylesheets/components/_divider.scss (hunk 3 above)
Comment (srsharon) [reply module/stylesheets/components/_divider.scss]: Done

**Review by jeradrutnam — APPROVED** [review]
Approved

File: module/index.js (hunk 2 above)
Comment (srsharon) [reply module/index.js]: Done

**Pull request merged**